src/JavaCodeGenerator.cs

```diff
@@ -1956,8 +1962,10 @@ public static JavaFile GetPageJavaFile(PageDetails pageClass, JavaSettings setti
 
         public static JavaFile GetXmlSequenceWrapperJavaFile(XmlSequenceWrapper xmlSequenceWrapper, JavaSettings settings)
         {
-            string xmlElementName = xmlSequenceWrapper.XmlElementName;
-            string xmlElementNameCamelCase = xmlElementName.ToCamelCase();
+            string xmlRootElementName = xmlSequenceWrapper.XmlRootElementName;
```

Comment:
Should these names be camel cased?

Reply:
No because these values are used as JsonProperty annotation arguments--if the swagger says the name of the root element is <foo> we have to be careful not to change it to <Foo>